sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/MessageData.cs

```diff
@@ -10,11 +10,12 @@ internal partial class MessageData
     {
         public MessageData(int version, LogRecord logRecord) : base(version)
         {
-            Message = LogsHelper.GetMessage(logRecord);
-            SeverityLevel = LogsHelper.GetSeverityLevel(logRecord.LogLevel);
-
             Properties = new ChangeTrackingDictionary<string, string>();
             Measurements = new ChangeTrackingDictionary<string, double>();
+
+            Message = LogsHelper.GetMessage(logRecord);
```

Comment:
from a perf standpoint, iterating once for message and iterating again for properties is inefficient. You can iterate just once, get props, and get message from {originalformat} in one go.